batch/batch/worker/worker.py

```diff
@@ -2178,9 +2178,9 @@ async def async_main():
 
     port_allocator = PortAllocator()
     network_allocator = NetworkAllocator()
-    await network_allocator.reserve()
 
     worker = Worker(httpx.client_session())
+    worker.task_manager.ensure_future(network_allocator.reserve())
```

Comment:
If there's an error in reserve, we'll never see it (we throw away the future). How much time do we spend waiting on this? Maybe do it in parallel with activating (i.e. `gather` the two tasks)?